src/datacatalog_tag_manager/datacatalog_facade.py

```diff
@@ -30,6 +30,21 @@ def create_or_update_tag(self, parent_entry_name, tag):
             logging.info('%sCreated: %s', self.__NESTED_LOG_PREFIX, created_tag.name)
             return created_tag
 
+    def delete_tag(self, parent_entry_name, tag):
+        entry_tags = self.__datacatalog.list_tags(parent=parent_entry_name)
+
+        try:
+            persisted_tag = next(
+                entry_tag for entry_tag in entry_tags
+                if entry_tag.template == tag.template and entry_tag.column == tag.column)
+            tag_name = persisted_tag.name
+            self.__log_operation_start('DELETE Tag: %s', tag_name)
+            self.__datacatalog.delete_tag(name=tag_name)
+            return tag_name
+        except StopIteration:
+            self.__log_operation_start('Tag not found for Tag Template: %s'
```

Comment:
Please replace `self.__log_operation_start` with `logging.error`. `__log_operation_start` adds blank and dashed lines to the log and is intended to work as the main operation delimiter (the main operation in this case is **DELETE Tag**  -- line 41).